PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/handler/IntelliJAzureExceptionHandler.java

```diff
@@ -70,9 +69,12 @@ public void handleException(Project object, Throwable throwable, boolean isBackG
 
     @Override
     protected void onHandleException(final Throwable throwable, final @Nullable AzureExceptionAction[] actions) {
-        final ProgressIndicator indicator = ProgressManager.getInstance().getProgressIndicator();
-        final boolean background = Objects.isNull(indicator) || !indicator.isModal();
-        onHandleException(throwable, background, actions);
+        Boolean backgrounded = AzureTaskContext.current().getBackgrounded();
```

Comment:
getBackgrounded ->  IsBackground, see https://docs.microsoft.com/en-us/dotnet/api/system.threading.thread.isbackground?view=net-5.0

Reply:
backgrounded -> background

Reply:
java thread does not has such a method like `isBackground`.
I use `backgrounded` since IntelliJ's `ProgressWindow` use it
![image](https://user-images.githubusercontent.com/69189193/103391719-cf784080-4b55-11eb-97e8-b4ad41d2b675.png)

the `getter` is `getBackgrounded` instead of `isBackgrounded` since my `backgrounded` is `Boolean` but not `boolean`.
and it's generated by `Lombok`